frontend/src/components/AutosaveIndicator.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+interface Props {
+  lastSaved: number | null;
+  isSaving: boolean;
+  userName: string;
+}
+
+const AutosaveIndicator: React.FC<Props> = ({ lastSaved, isSaving, userName }) => {
+  const style = {
+    font: "noto-serif",
+    position: 'absolute' as 'absolute',
+    top: '100',
+    left: '152',
+    transform: 'translateY(-65%)', // Move it above the title
+    color: '#888',
+    fontSize: '13px',
+    display: 'flex',
+    alignItems: 'center',
+    gap: '8px',
+    whiteSpace: 'nowrap' as 'nowrap', // Prevent text from wrapping
```

Comment:
use tailwind instead of css.
Also, try to implement this without using absolute.